sdk/monitor/azure-monitor-query/migration_guide.md

```diff
@@ -0,0 +1,110 @@
+# Guide for migrating from azure-loganalytics 0.1.0 to azure-monitor-query 1.0.x
+This guide is intended to assist in the migration to `azure-monitor-query` 1.0.x from `azure-loganalytics` v0.1.0.
+It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the `azure-loganalytics` v0.1.0 package is assumed.
+For those new to the Monitor Query client library for Python, please refer to the [README for `azure-monitor-query`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-query/README.md) rather than this guide.
```

Comment:
```suggestion
Familiarity with the `azure-loganalytics` v0.1.0 package is assumed. If you're new to the Azure Monitor Query client library for Python, see the [README for `azure-monitor-query`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-query/README.md) instead of this guide.
```